### PR TITLE
Upgrade prettier to v2.7.1

### PR DESCRIPTION
**Pull request by krispenney**
This PR updates prettier to v2.7.1! The biggest change that I made was to the glob patterns in the various package.json files, these weren't actually matching anything anyways. We may as well let prettier examine all files in these subdirectories anyways.

### CODE REVIEW

File: packages/@misk/simpleredux/src/utilities/index.ts [hunk 1]
```diff
@@ -139,7 +139,7 @@ export const getFirstTag = <
   [key: string]: UNIONED_TYPE
 }): T => {
   if (Object.keys(payload).length === 1) {
-    return (payload[Object.keys(payload)[0]] as unknown) as T
```
Comment (keeferrourke) [inline packages/@misk/simpleredux/src/utilities/index.ts:142]: sorta wish we could keep these clarifying parens

**Review by keeferrourke — APPROVED** [review]
LGTM

File: packages/@misk/simpleredux/src/utilities/index.ts (hunk 1 above)
Comment (krispenney) [reply packages/@misk/simpleredux/src/utilities/index.ts:142]: hmm I can see if there's a rule to keep these around

Comment (krispenney) [reply packages/@misk/simpleredux/src/utilities/index.ts:142]: There's not really a way to configure this specifically (prettier is very opinionated regarding it's style). However, I've rewritten this line in a way that I think is clearer to begin with!

**Comment by adrw** [comment]
I'd like to provide some feedback on these changes, specifically the glob change. Notably, this now has Prettier run on the generated code in the `lib` directory. 

This both directly increases the compile time wasted linting these generated files and also may have unintentional consequences changing the minified code to be inflated in size from formatting.

Could the glob changes be reverted and these considerations kept in mind going forward?

**Pull request merged**